### PR TITLE
Add test suite description for bug reporting

### PR DESCRIPTION
**Pull request by jknphy**
#### Summary: 
Add test suite description for bug reporting.
#### Related ticket:
https://progress.opensuse.org/issues/45890
#### Verification run: 
For example, try to open a bug in my openQA instance for this step: [](http://10.100.12.156/tests/2847#step/addon_products_via_SCC_yast2/37)

### CODE REVIEW

File: lib/OpenQA/Schema/Result/Jobs.pm [hunk 1]
```diff
@@ -293,6 +293,12 @@ sub scenario {
     return $scenario;
 }
 
+sub scenario_description {
+    my ($self) = @_;
+    my $description = $self->result_source->schema->resultset('TestSuites')->find({ name => $self->TEST})->description;
```
Comment (Martchus) [inline lib/OpenQA/Schema/Result/Jobs.pm]: DBIx has better means to do that query. Rather than explaining I can push a further commit to your branch.

Comment (jknphy) [reply lib/OpenQA/Schema/Result/Jobs.pm]: Sure, :) I finally found some solution after looking at the tables, but if you know something better, go ahead.

**Comment by jknphy** [comment]
I'm thinking to add a subtest to `10-jobs.t` but trying to figuring out what could be the minimum to call the function there.

**Comment by Martchus** [comment]
@jknphy Don't worry, I've already added a test and fixed the bug causing the error pages test to fail. I also improved the DBIx line like I said. So the CI tests should pass now.

**Comment by jknphy** [comment]
very clean code! I learning with your commits. Thanks! :)

**Comment by Martchus** [comment]
Seems like tests are not passing. Likely just missing cascade delete.

**Comment by jknphy** [comment]
Perhaps adding `scenario => 'Installation in textmode and selecting the textmode "desktop" during installation.'` after https://github.com/os-autoinst/openQA/blob/master/t/ui/16-tests_job_next_previous.t#L61

**Comment by Martchus** [comment]
It was just the missing `{cascade_delete => 0}` in the relationship and the name `scenario` was already used (so I renamed the previously present `scenario` to `scenario_name`).

**Pull request merged**